rialto_airflow/harvest/pubmed.py

```diff
@@ -104,6 +149,24 @@ def pmids_from_orcid(orcid):
     return _pubmed_search_api(f"{orcid}[auid]")
 
 
+def pmids_from_dois(dois: list[str]) -> list[str]:
+    """
+    Returns PMIDs associated with given list of DOIs.
+    """
+
+    # Note that we need to do these one by one, since Pubmed doesn't support searching for multiple DOIs
+    pmids = []
+    for doi in dois:
+        # look up the PMID given a DOI
+        pmid_results = _pubmed_search_api(f"{doi}")
```

Comment:
If you aren't formatting the doi you can just pass it in as a string?

```python
_pubmed_search_api(doi)
```

Reply:
yes good catch, this was leftover from when i was actually using formatting to do something more complicated, but no longer needed